--- /Users/unity/Documents/GitHub/com.unity.uiwidgets/engine/Build.bee.cs	2021-05-17 10:44:45.000000000 +0800
+++ /Users/unity/Documents/GitHub/com.unity.uiwidgets/engine/Scripts/Build.bee.cs	2021-05-17 10:44:20.000000000 +0800
@@ -751,7 +751,7 @@
         //SetupFml(np);
         //SetupSkia(np);
         //SetupTxt(np);
-        var codegens = new[] { CodeGen.Release };
+        var codegens = new[] { CodeGen.Debug };
         dependencies = new List<NPath>();
 
         if (platform == UIWidgetsBuildTargetPlatform.windows)
